topmenubar/src/main/java/de/interseroh/tmb/client/TopMenuBarWebApp.java

```diff
@@ -43,12 +47,22 @@
 	private static final String DATA_TMB_HEADLINE = "data-tmb-headline";
 	private static final String DEFAULT_BACKGROUND_COLOR = "#FF0000";
 	private static final String TOP_MENU_BAR_PLACEHOLDER = "tmb_top_menu_bar";
+	private static final String TOP_MENU_ICONS_RIGHT = "tmb_icons_right";
+
+	private static final String PORTAL_LINKS = "tmb_portal_links";
+	private static final String TOPICS = "internal_topics";
+
+	private static final String COLLAPSEID = "tmb_navbar_collapse";
 	private static final Logger logger = Logger
 			.getLogger(TopMenuBarWebApp.class.getName());
 	// Create Gin Injector
 	private final TopMenuBarAppGinjector injector = GWT
 			.create(TopMenuBarAppGinjector.class);
 
+	private RootPanel portal;
+	private RootPanel topics;
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Rename field "topics" to prevent any misunderstanding/clash with field "TOPICS" defined on line 53. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1845)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the "topics" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1450)